src/main/resources/org/jenkinsci/plugins/xunit/types/boosttest-1.5.0.xsd

```diff
@@ -157,6 +157,7 @@ THE SOFTWARE.
             <xs:choice minOccurs="0" maxOccurs="unbounded">
                 <xs:element ref="TestCase" minOccurs="0" maxOccurs="unbounded"/>
                 <xs:element ref="TestSuite" minOccurs="0" maxOccurs="unbounded"/>
+                <xs:element ref="Message" minOccurs="0" maxOccurs="unbounded"/>
```

Comment:
@spoorcc, shouldn't this new field be `optional` for backward compatibility?

Reply:
minOccurs=0 is effectvly optional.
I trying to run the build using the XSD and ouput log reported in JENKINS-42031 by Thomas Archer and there are not failure on other test case where Message is not present as child of TestSuite.